prow/config/tide.go

```diff
@@ -78,6 +83,61 @@ func (t *Tide) MergeMethod(org, repo string) github.PullRequestMergeType {
 	return v
 }
 
+// TideBranchProtection is initially used to know whether to use the github status or only check for required checks.
+// This can extended in the future to support
```

Comment:
unfinished sentence

Reply:
How do I configure the required checks? If this is something that is done on Github alongside configuring branch protection, please add pointers to documentation on how to do it.